juno_magic/extensions/wamp.py

```diff
@@ -222,6 +224,9 @@ def __init__(self, shell):
 
         self._parser = self.generate_parser()
 
+        self._logger = log
+        self._logger.startLogging(open("juno-magic-logger"), "w")
```

Comment:
Don't think a hard coded, relative file path is appropriate here.  Also this is a reference to the global twisted log right? So wouldn't it log any and all twisted code?  I suspect we need a better / different solution for this.  Any ideas?